src/HttpClientFactory/Polly/src/.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.{cs,vb}]
+
+# CA2007: Consider calling ConfigureAwait on the awaited task
+dotnet_diagnostic.CA2007.severity = warning
```

Comment:
Could we leave this file out? I'd like to see if we can enable this rule at the global level and make it a non-warning for projects that are part of ASP.NET Core / reference the shared fx?

Reply:
I can take it out if you like - I'd added it based on https://github.com/dotnet/aspnetcore/pull/39865#issuecomment-1024945726

Reply:
Yes please. I might give it a go this week. FYI @BrennanConroy

Reply:
> if we can enable this rule at the global level and make it a non-warning for projects that are part of ASP.NET Core / reference the shared fx

Sounds good, didn't know you were planning on doing this :+1: